src/Microsoft.DotNet.CodeFormatting/IFormattingEngine.cs

```diff
@@ -16,7 +16,7 @@ public interface IFormattingEngine
         ImmutableArray<string> FileNames { get; set; }
         ImmutableArray<IRuleMetadata> AllRules { get; }
         ImmutableArray<DiagnosticDescriptor> AllSupportedDiagnostics { get; }
-        bool AllowTables { get; set; }
+        bool AllowTables { set; }
```

Comment:
oops. was looking at dependencies on the write. this property probably warrants a rename but I want to verify with @jared or @Priya91 that I get what it's used for.